### PR TITLE
Commercial: remove the BestBuy component

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Confirmed as no longer used.

cc @guardian/commercial-dev

### CODE REVIEW

File: common/app/common/configuration.scala [hunk 1]
```diff
@@ -318,8 +318,6 @@ class GuardianConfiguration extends Logging {
     lazy val liveEventsToken = configuration.getStringProperty("live-events.token")
     lazy val liveEventsMembershipUrl = "https://membership.theguardian.com/events.json"
     lazy val jobsUrl= configuration.getStringProperty("jobs.api.url")
-    lazy val mortgagesUrl = configuration.getStringProperty("lc.mortgages.api.url")
-    lazy val moneyUrl = configuration.getStringProperty("moneysupermarket.api.url")
```
Comment (kelvin-chappell) [inline common/app/common/configuration.scala]: Should remove these properties from `platform` as well.

**Comment by rich-nguyen** [comment]
super awesome! 👍

File: static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js [hunk 2]
```diff
@@ -32,7 +30,6 @@ define([
     reduce
 ) {
     var urlBuilders = {
-        bestbuy:        defaultUrlBuilder('money/bestbuys'),
```
Comment (kelvin-chappell) [inline static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js]: There will presumably be a creative template in DFP that uses this endpoint, which should be removed or disabled.

File: common/app/common/configuration.scala (hunk 1 above)
Comment (regiskuckaertz) [inline common/app/common/configuration.scala]: Will submit a PR for that too, thanks!

File: static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js (hunk 2 above)
Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js]: Yep, doing that.

**Comment by kelvin-chappell** [comment]
🔥 👍

**Pull request merged**